actionpack/lib/action_dispatch/testing/assertions/routing.rb

```diff
@@ -19,7 +19,7 @@ module RoutingAssertions
       #
       # You can also pass in +extras+ with a hash containing URL parameters that would normally be in the query string. This can be used
       # to assert that values in the query string will end up in the params hash correctly. To test query strings you must use the
-      # extras argument, appending the query string on the path directly will not work. For example:
+      # extras argument. Appending the query string on the path directly will not work. For example:
```

Comment:
Maybe this sentence would be better as "To test query strings you must use the extras argument because appending the query string on the path directly will not work." - splitting it in two confuses the meaning.